ivy/functional/frontends/paddle/fft.py

```diff
@@ -44,3 +44,11 @@ def fftshift(x, axes=None, name=None):
     roll = ivy.roll(x, shifts, axis=axes)
 
     return roll
+@with_supported_dtypes(
+    {"2.5.0 and below": ("complex64", "complex128")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def fft2(x, n=None, axes=(- 2, - 1), norm='backward', name=None):
+    ret = ivy.fft2(ivy.astype(x, "complex128"), axes, norm=norm, n=n)
```

Comment:
If you have any specific reason for converting to `complex128`, let me know. Also `ivy.fft2` doesn't take `n` as an argument.

Reply:
In that case, can you remove these type casting to avoid overheads?